torch/quantization/quantize.py

```diff
@@ -353,6 +353,12 @@ def swap_module(mod, mapping):
     if hasattr(mod, 'qconfig') and mod.qconfig is not None or type(mod) == DeQuantStub:
         if type(mod) in mapping:
             new_mod = mapping[type(mod)].from_float(mod)
+            # Preserve the module pre and post forward hooks except _observer_forward_hook
```

Comment:
could you also write down when the hooks will be executed? that is: before/after quantized tensor

Reply:
Updated